web/packages/teleport/src/components/LabelsInput/LabelsInput.tsx

```diff
@@ -146,46 +146,42 @@ export function LabelsInput({
   const width = `${inputWidth}px`;
   const inputSize = 'medium';
   return (
-    <Fieldset gap={labels.length > 0 ? 2 : 1}>
-      <Stack gap={1}>
```

Comment:
I was wondering how it broke the semantic structure of fieldset and legend, I managed to find that indeed:

> The first element inside the fieldset must be a legend element, which provides a label or description for the group.

https://www.w3.org/TR/WCAG20-TECHS/H71.html

FWIW, I did a quick check and this could also be fixed by getting rid of this `Stack` and changing the gap on `Fieldset` to always be 2, with just a minimal difference in spacing compared to the original implementation while keeping spacing defined on parents and not individual children.